src/kakarot/accounts/eoa/library.cairo

```diff
@@ -52,6 +56,23 @@ namespace ExternallyOwnedAccount {
         data_len: felt,
     }
 
+    func get_eth_address{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (
+        address: felt
+    ) {
+        let address = eth_address.read();
+        return (address);
+    }
+
+    func set_eth_address{
```

Comment:
I don't think that we want to have a setter since it's supposed to be immutable as the starknet address is